tests/foreman/cli/test_repository.py

```diff
@@ -361,8 +523,15 @@ def test_positive_update_url(self):
         @Assert: Repository url is updated
         """
         new_repo = self._make_repository()
+        # generate a tuple of auth repos with all valid credentials
+        auth_repos = ()
```

Comment:
+ you are not simply added one more repo to the scope, but adding one more loop
```
+        for repo in (FAKE_5_YUM_REPO, FAKE_7_PUPPET_REPO):
 +            auth_repos += tuple(
 +                repo.format(creds['login'], creds['pass'])
 +                for creds in valid_http_credentials(url_encoded=True))
```
which doesn't have any connection to the old logic and if it fail due any reason we will not be able to verify things that we have before.
Even that one fact should force us to separate cases on two